beacon-chain/node/node.go

```diff
@@ -17,6 +17,9 @@ import (
 	"sync"
 	"syscall"
 
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/db/pruner"
+	"github.com/prysmaticlabs/prysm/v5/time/slots"
```

Comment:
Nit, I dont think this aligns with prysm repo's import rules

Reply:
Fixed, do we have any linter to pick this up?